### PR TITLE
MVPoly: implement test prime::dense from expr

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: mvpoly/tests/prime.rs [hunk 1]
```diff
@@ -353,3 +357,139 @@ fn test_from_variable_column() {
     assert_eq!(p[3], Fp::zero());
     assert_eq!(p[4], Fp::one());
 }
+
+#[test]
+/// As a reminder, here the equations to compute the addition of two different
+/// points `P1 = (X1, Y1)` and `P2 = (X2, Y2)`. Let define `P3 = (X3, Y3) = P1 +
```
Comment (Fizzixnerd) [inline mvpoly/tests/prime.rs:363]: NitPick: suggest: "here _are_ the equations"

Comment (dannywillems) [reply mvpoly/tests/prime.rs]: Done.

File: mvpoly/tests/prime.rs [hunk 2]
```diff
@@ -453,3 +457,166 @@ fn test_eval_pbt_neg() {
...
+        let eval = p.eval(&random_evaluation);
+        let exp_eval = {
+            random_evaluation[5] + random_evaluation[1] + random_evaluation[2]
+                - random_evaluation[0] * random_evaluation[0]
+        };
+        assert_eq!(eval, exp_eval);
+    }
+    {
+        // - Constraint 3: Y3 - λ (X1 - X3) - Y1 = 0
+        let expr = y3.clone() - lambda.clone() * (x1.clone() - x3.clone()) + y1.clone();
```
Comment (Fizzixnerd) [inline mvpoly/tests/prime.rs:611]: equation does not match the expression.

**Review by Fizzixnerd — APPROVED** [review]
pending small change above, approved.

File: mvpoly/tests/prime.rs (hunk 2 above)
Comment (dannywillems) [reply mvpoly/tests/prime.rs:611]: Addressed in https://github.com/o1-labs/proof-systems/pull/2505

**Pull request merged**